packages/pipcook-app/src/models/textClassification.ts

```diff
@@ -0,0 +1,73 @@
+
+import {DataCollect, DataAccess, ModelLoad, ModelTrain, ModelEvaluate, PipcookRunner, ModelDeploy, downloadZip} from '@pipcook/pipcook-core';
+
+import textClassDataCollect from '@pipcook/pipcook-plugins-text-class-data-collect';
+import textClassDataAccess from '@pipcook/pipcook-plugins-text-csv-data-access';
+import bayesianClassiferModelLoader from '@pipcook/pipcook-plugins-bayesian-classifier-model-load';
+import bayesianClassifierModelTrain from '@pipcook/pipcook-plugins-bayesian-classifier-model-train';
+import classModelEvalute from '@pipcook/pipcook-plugins-bayesian-classifier-model-evaluate';
+import textClassLocalModelDeploy from '@pipcook/pipcook-plugins-text-class-local-model-deploy';
+import textClassEasDeploy from '@pipcook/pipcook-plugins-bayesian-classifier-model-eas-deploy'
+
+import { getEasParam, EasConfigI } from '../utils/utils';
```

Comment:
```suggestion
import {getEasParam, EasConfigI} from '../utils/utils';
```

Consistent with the aboves.